shunit2

```diff
@@ -630,7 +630,12 @@ failNotEquals() {
   shunit_actual_=$2
 
   shunit_message_=${shunit_message_%% }
-  _shunit_assertFail "${shunit_message_:+${shunit_message_} }expected:<${shunit_expected_}> but was:<${shunit_actual_}>"
+  if command [ $(echo "${shunit_expected_}" | wc -l) -gt 10 ] || command [ $(echo "${shunit_actual_}" | wc -l) -gt 10 ]; then
```

Comment:
Could you wrap this line just before the ||, and place the "then" on its own line, so that the 80col line limit is honoured?
https://github.com/kward/shunit2/wiki/Coding-standards#wrapping

Reply:
Done.